packages/app/src/cli/services/context.ts

```diff
@@ -884,18 +914,11 @@ export function renderCurrentlyUsedConfigInfo({
   resetMessage,
   includeConfigOnDeploy,
 }: CurrentlyUsedConfigInfoOptions): void {
-  const items = [`App:             ${appName}`]
-
-  if (org) items.unshift(`Org:             ${org}`)
-  if (devStore) items.push(`Dev store:       ${devStore}`)
-  if (updateURLs) items.push(`Update URLs:     ${updateURLs}`)
-  if (includeConfigOnDeploy !== undefined) items.push(`Include config:  ${includeConfigOnDeploy ? 'Yes' : 'No'}`)
-
-  let body: TokenItem = [{list: {items}}]
-  if (resetMessage) body = [...body, '\n', ...resetMessage]
+  const devStores = []
+  if (devStore) devStores.push(devStore)
 
+  const body = formInfoBoxBody(appName, org, devStores, resetMessage, updateURLs, includeConfigOnDeploy)
   const fileName = (appDotEnv && basename(appDotEnv)) || (configFile && getAppConfigurationFileName(configFile))
-
   renderInfo({
```

Comment:
This makes sense to me, but we should get somebody from the CLI side to take a peek since we're refactoring their code